web-server/src/components/GithubButton.tsx

```diff
@@ -0,0 +1,57 @@
+import StarBorderOutlined from '@mui/icons-material/StarBorderOutlined';
+import { Button, ButtonProps, Divider, useTheme } from '@mui/material';
+import { FC, useMemo } from 'react';
+
+import { FlexBox } from '@/components/FlexBox';
+import { Line } from '@/components/Text';
+import { useSelector } from '@/store';
+
+const githubRepoUrl = `https://github.com/middlewarehq/middleware`;
+
+const formatStarCount = (stars: number) => {
+  if (isNaN(stars)) return '';
+  if (stars < 1000) return `${stars}`;
+
+  let digit = `${(stars / 1000).toFixed(1)}`;
+  if (digit.endsWith('0')) digit = digit.slice(0, -2);
+
+  return `${digit}k`;
+};
```

Comment:
I'll give you a super neat way to show numbers rounded to a fixed number of decimals, but without any trailing zeroes such that they show up like:


✅ 1.23
✅ 4.5
✅ 6

And not:
❌ 1.230
❌ 4.50 or 4.500
❌ 6.0 or 6.00 or 6.000

---
```
const num = Number(myNum.toFixed(2));
                                 ^ however many decimal places
                                   should be shown, maximum,
                                   else rounded off
```

In fact, you're almost there!

```suggestion
const formatStarCount = (stars: number) => {
  if (isNaN(stars)) return '';
  if (stars < 1000) return `${stars}`;

  return Number((stars / 1000).toFixed(1));
};
```

Reply:
@jayantbh In line 15, k is missing.

Reply:
Oh yeah, lol. Yeah do add just that. Good eye. 👓